DEVELOPING.md

```diff
@@ -0,0 +1,41 @@
+# Development guides and tips  
+
+## Linting
+
+### Run `golangci-lint`
+
+Full list linters and config can be found in `.golangci.yml`
+
+- Consider running the linter locally before commit.  
```

Comment:
Actually, I would suggest instead changing this to indicate a more stronger recommendation/requirement:

Developers are expected to run `make ci-job` prior to posting PRs. This will run the various linters, as well as required unit tests. The golangci-lint test can be run individually at any time during development, as well, with the `make golangci-lint` job to scan code changes.